### PR TITLE
Remove IntersperseMultipleStoryPackages a/b test

### PR DESCRIPTION
**Pull request by TBonnin**
Remove the IntersperseMultipleStoryPackages A/B testing
We are going forward with intermixing stories when there are more than
one package.
## Does this affect other platforms - Amp, Apps, etc?

No
## Request for comment

@piuccio @johnduffell

### CODE REVIEW

File: common/app/mvt/MultiVariateTesting.scala [hunk 1]
```diff
@@ -132,14 +117,7 @@ object MultiVariateTesting {
...
 
-  // buckets 0-7 are removed because they cost $1000+ just in aws bandwidth every month, the rest
-  // will be removed once they're not in use.  In future server side ab tests will be added explicitly
-  // to target only the URLs needed for the time needed.
-  object Variant8 extends Variant("variant-8")
-  object Variant9 extends Variant("variant-9")
-
-  private val allVariants = List(
-    Variant8, Variant9)
+  private val allVariants = List[Variant]()
```
Comment (TBonnin) [inline common/app/mvt/MultiVariateTesting.scala]: @johnduffell I will follow up in the next few days with another PR to refactor and clean this `MultiVariateTesting`. I didn't want to mix all those changes here

File: applications/app/controllers/GalleryController.scala [hunk 2]
```diff
@@ -43,7 +43,7 @@ object GalleryController extends Controller with RendersItemResponse with Loggin
       .showFields("all")
     ).map { response =>
       val gallery = response.content.map(Content(_))
-      val model: Option[GalleryPage] = gallery collect { case g: Gallery => GalleryPage(g, RelatedContent(g, response), index, isTrail) }
+      val model: Option[GalleryPage] = gallery collect { case g: Gallery => GalleryPage(g, StoryPackages(g, response), index, isTrail) }
```
Comment (TBonnin) [inline applications/app/controllers/GalleryController.scala]: In my previous change I had anticipated the rename of the `RelatedContent` object into `StoryPackages` by duplicating some code. See https://github.com/guardian/frontend/commit/0425a079e3af26b61c3fb1b36cd243312ca511fa#diff-9b3d36cd982208831cf027d918e19a9cL26

Now I am cleaning up the 2 different a/b versions, I can go forward with this renaming (which make more sense)

File: common/app/model/RelatedContent.scala [hunk 3]
```diff
@@ -15,28 +15,11 @@ case class RelatedContentItem (
   faciaContent: PressedContent
 )
 
-case class RelatedContent ( items: Seq[RelatedContentItem], storyPackagesCount: Int = 0) {
```
Comment (TBonnin) [inline common/app/model/RelatedContent.scala]: `storyPackagesCount` is not useful anymore. It was only used here https://github.com/guardian/frontend/pull/12938/files#diff-79e70845e1ad76d8d445887a5498a2fdL15

**Comment by piuccio** [comment]
looks good to me, but I'm no scala expert 👍

**Pull request merged**